README.md

```diff
@@ -102,45 +39,113 @@ Cryptol](INSTALL.md).
-   cryptographic engine. Success here requires use of modules and concepts
-   from many of the previous labs.
-
-
-## Graphical View of the Course
-
-The suggested flow is to follow the red line. The black lines indicate
-labs designed to give you more opportunities to practice Cryptol, but
-are not strictly necessary for course completion.
-
-<img class="center" src="https://raw.githubusercontent.com/weaversa/cryptol-course/master/misc/deps.svg" alt="Dependencies and Suggested Course Flow">
+9. [Capstone](labs/LoremIpsum/LoremIpsum.md): Use components and
+   techniques from other labs to decrypt a series of secret messages
+   by feeding wrapped keys into the anomalous KLI20 cryptographic
+   engine. 
 
-## Extra Labs and Demonstrations
+### Extra Labs and Demonstrations
 
-There are also labs that go beyond the scope of the course outline, but
+There are also modules that go beyond the scope of the course outline but
 may prove interesting or useful for students with wider interests or
 different backgrounds.
 
-1. [Arithmetic Verifications](labs/saw/ArithmeticVerifications/ArithmeticVerifications.md):
+1. [Arithmetic
+   Verifications](labs/saw/ArithmeticVerifications/ArithmeticVerifications.md):
    This lab demonstrates using SAW to prove facts about various
    arithmetic implementations. Some simple reference examples are
-   included as well as an illustration of a barrier one may experience
-   using these techniques.
+   included as well as an illustration of a barrier one may encounter
+   when using these techniques.
+
+2. [Bit-twiddling](labs/saw/bittwiddling/bittwiddling.md): This lab
+   contains a collection of famous, yet hard to understand algorithms
+   which perform relatively simple calculations. This is an
+   introduction to how one can use Cryptol and SAW to prove that an
+   implementation matches its specification.
+
+The general course flow is represented by the red line. The black lines
+indicate modules designed to give you more opportunities to practice
+Cryptol, but are not strictly necessary for course completion.  (Click
+on the image below for a navigable representation.)
+
+<img class="center"
+src="https://raw.githubusercontent.com/weaversa/cryptol-course/master/misc/deps.svg"
+alt="Dependencies and Suggested Course Flow">
+
+## How To Take This Course
+
+This modules of this course are monolithic
+[literate](https://en.wikipedia.org/wiki/Literate_programming) Cryptol
+documents with a `.md` suffix: each file contains both exposition
+about the topic at hand as well embedded lab exercises.  When viewed
+via a web browser (either on the [GitHub cryptol
+course](https://github.com/saweave/cryptol-course) site or via a local
+mechanism), the file will be rendered like this:
+
+<hr>
+...
+
+### Functions
+
+Functions in Cryptol can include input parameters, perform operations
+on those parameters, and generate a result.  
+
+**EXERCISE:** Create the body of the function below.
+
+```comment
```

Comment:
I like restructuring but I really want a crypto centric example. Integers stay away!

Reply:
Okay, but please something more palatable (read: less intimidating).

Reply:
Any suggestions? CBC mode is a one-liner with a nice property.

Reply:
Simple bit-oriented example put in its place.